lib/downloader.rb

```diff
@@ -25,32 +26,47 @@ def setTermSize
   return true
 end
 
-def downloader (*args)
+def downloader (url, sha256sum, filename = File.basename(url), retry_count = 0, verbose = false)
+  # downloader: wrapper for all Chromebrew downloaders (`net/http`,`curl`...)
+  # Usage: downloader <url>, <filename::optional>, <retry_count::optional>,
+  #                   <verbose::optional>, sha256sum: <optional>
```

Comment:
Change usage to mention sha256sum check is required?

Reply:
Done